src/topology/algebra/module/multilinear.lean

```diff
@@ -174,7 +174,7 @@ end has_continuous_add
 /-- If `f` is a continuous multilinear map, then `f.to_continuous_linear_map m i` is the continuous
 linear map obtained by fixing all coordinates but `i` equal to those of `m`, and varying the
 `i`-th coordinate. -/
-def to_continuous_linear_map (m : Πi, M₁ i) (i : ι) : M₁ i →L[R] M₂ :=
+def to_continuous_linear_map [decidable_eq ι] (m : Πi, M₁ i) (i : ι) : M₁ i →L[R] M₂ :=
```

Comment:
Why would you need `decidable_eq` in this one?

Reply:
For `continuous_const.update`; you need to check which vector is `i`.

Reply:
Sure, but the definition is an implementation detail which should not be unfolded. We could use the classical instance in the definition of `to_fun` instead, and register a lemma that under `[decidable_eq iota]` then the coercion is what we expect (using this generic instance). I'll let you decide.